ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -311,6 +319,18 @@ def _get_matching_session_infos(self, instrument_type_id: str) -> List[SessionIn
             info for info in self._session_info if instrument_type_id == info.instrument_type_id
         ]
 
+    def _try_get_matching_multiplexer_session_info(
```

Comment:
We have been reserving the prefix "try_" for methods that implement the .NET Try pattern, which involves returning a bool indicating whether the operation succeeded.

In this case, I'd rather remove "try_" and call this "_get_matching_multiplexer_session_info"